core/src/main/java/org/apache/accumulo/core/util/cleaner/CleanerUtil.java

```diff
@@ -107,10 +107,14 @@ public static Cleanable shutdownThreadPoolExecutor(Object tpe, Logger log) {
     requireNonNull(tpe);
     requireNonNull(log);
     return CLEANER.register(tpe, () -> {
+      ThreadPoolExecutor pool = (ThreadPoolExecutor) tpe;
+      if (pool.isShutdown()) {
+        return;
+      }
```

Comment:
The other cleanables pass in the closed atomic boolean, so they can detect whether the resource has already been closed, and the cleanable action isn't needed. I suggest following the same pattern, just for convenience, rather than follow a custom pattern here.